BUILDING.md

```diff
@@ -0,0 +1,128 @@
+# Building Dark Reader
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+
+## Microsoft Windows instructions
+
+_Microsoft Windows 7 or later is required_
+
+### Installing prerequisites via Chocolatey
+
+_Chocolatey installation steps based on [chocolatey.org/install](https://chocolatey.org/install)_
```

Comment:
Just let them figure out how to install chocolatey by referring to the installation page, as well as the installation page is the exact content what's here. So I don't see why we are copy pasting the steps here which can be out of date by tomorrow.

Reply:
38fd8eb